gpcontrib/arenadata_toolkit/src/include/bloom.h

```diff
@@ -0,0 +1,40 @@
+#ifndef BLOOM_H
+#define BLOOM_H
+
+#include "postgres.h"
+
+#include <stdint.h>
+
+#define MAX_BLOOM_HASH_FUNCS 6
+#define TOTAL_ELEMENTS 10000000UL
+
+typedef struct
+{
+	uint8	   *current_bloom;
+	uint32		size;			/* equal to bloom_size, half a map size */
+	uint8		is_set_all;		/* indicates all bits are set */
+	uint8		map[FLEXIBLE_ARRAY_MEMBER]; /* core bit array */ ;
+}	bloom_t;
+
+static inline Size
+full_bloom_size(uint32 size)
+{
+	return (offsetof(bloom_t, map) + size);
```

Comment:
Shouldn't it be something like that? According to the description of the PR.
```suggestion
	return (offsetof(bloom_t, map) + size * 2);
```

Reply:
No, size is taken from function arguments. This function is also used for temp filter like local filter in track function or uncommited_bloom.